tests/actions/ReportTest.ts

```diff
@@ -52,12 +53,12 @@ describe('actions/Report', () => {
     afterEach(PusherHelper.teardown);
 
     it('should store a new report action in Onyx when onyxApiUpdate event is handled via Pusher', () => {
-        global.fetch = TestHelper.getGlobalFetchMock();
+        global.fetch = TestHelper.getGlobalFetchMock() as typeof fetch;
```

Comment:
```suggestion
        // @ts-expect-error TODO: Remove this once TestHelper (https://github.com/Expensify/App/issues/25318) is migrated to TypeScript.
        global.fetch = TestHelper.getGlobalFetchMock();
```

I think it's best to just suppress the error until that file is migrated.